src/druidry/data_source.py

```diff
@@ -322,6 +322,13 @@ def combine_filter(f):
         return OrFilter(fields=filters)
 
 
+def negate_filter(f):
+    filter_ = translate_filter(f['filter'])
```

Comment:
https://realpython.com/python-pep8/#naming-styles
let's try for something like `translated_filter`

Reply:
fixed :) I was trying to follow naming convention in other functions in the same file as well as [https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)